doc/zeek.rst

```diff
@@ -409,6 +409,8 @@ As a full example, here's what a new GIF analyzer could look like:
         parse with GIF::Image,
         mime-type image/gif;
 
+.. _zeek_events:
```

Comment:
Just as a heads-up: While adding anchors to the FAQ rubrics I noticed that Sphinx automatically adds anchors derived from the text to headings (but not rubrics), this one would be `#event-definitions`. This is insufficient if we need to refer to sections with identical title, but often good enough.

Reply:
Yeah, I know about that but I don't like to use it because it's so brittle: changing the header not only breaks internal links that then need fixing, but also invalidates external links to that section existing elsewhere.